test/integration/projects_int_test.go

```diff
@@ -75,6 +77,87 @@ func (suite *ProjectsIntegrationTestSuite) TestJSON() {
 	//AssertValidJSON(suite.T(), cp) // list is too large to fit in terminal snapshot
 }
 
+func (suite *ProjectsIntegrationTestSuite) TestEdit_Name() {
+	suite.OnlyRunForTags(tagsuite.Projects)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
+
+	// What we expect the project name to be and what we want to change it to.
+	// This can change if the test failed previously.
+	var (
+		originalName = "Edit-Test"
+		newName      = "Edit-Rename"
+	)
+
+	cp := ts.Spawn("checkout", fmt.Sprintf("ActiveState-CLI/%s", originalName))
+
+	// If the checkout failed, it's probably because the project name was changed
+	// in a previous run of this test. Try again with the new name.
+	if strings.Contains(cp.TrimmedSnapshot(), "Could not checkout project") {
+		cp = ts.Spawn("checkout", fmt.Sprintf("ActiveState-CLI/%s", newName))
+		originalName = newName
+		newName = originalName
+	}
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("projects")
+	cp.Expect(originalName)
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("projects", "edit", fmt.Sprintf("ActiveState-CLI/%s", originalName), "--name", newName)
+	cp.Expect("You are about to edit")
+	cp.Send("y")
+	cp.Expect("Project edited successfully")
+	cp.ExpectExitCode(0)
+
+	// Verify the local checkouts have been updated
+	cp = ts.Spawn("projects")
+	cp.Expect(newName)
+	cp.ExpectExitCode(0)
+
+	// Change name back to original
+	cp = ts.Spawn("projects", "edit", fmt.Sprintf("ActiveState-CLI/%s", newName), "--name", originalName)
+	cp.Expect("You are about to edit")
+	cp.Send("y")
```

Comment:
After supporting non-interactive mode as mentioned earlier, I suggest changing this invocation to utilize it.

Reply:
The ACs in the ticket state that we should test the interactive scenarios.